Breaking vite-plugin-node-polyfills v0.23.1

The latest version of `vite-plugin-node-polyfills@0.23.1` which aim to support vite v7 is actually broken - i.e. the E2E tests are failing because it cannot starts the applications.

See CI https://github.com/junobuild/create-juno/actions/runs/16101329026/job/45430864009?pr=151

Locally I get:

> (!) Failed to run dependency scan. Skipping dependency pre-bundling. Error: Context failed with 1 error:
/Users/daviddalbusco/projects/juno/create-juno/templates/vue-starter/node_modules/resolve/lib/async.js:146:34: ERROR: [plugin: node-stdlib-browser-alias] Cannot find module 'unenv/mock/empty' from '.'

Related issue: https://github.com/davidmyersdev/vite-plugin-node-polyfills/issues/133

--- a/templates/react-example/package.json
+++ b/templates/react-example/package.json
@@ -38,6 +38,6 @@
     "prettier-plugin-tailwindcss": "^0.6.13",
     "tailwindcss": "^4.1.10",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0"
+    "vite-plugin-node-polyfills": "^0.24.0"
   }
 }

--- a/templates/react-starter/package.json
+++ b/templates/react-starter/package.json
@@ -37,6 +37,6 @@
     "prettier-plugin-tailwindcss": "^0.6.13",
     "tailwindcss": "^4.1.10",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0"
+    "vite-plugin-node-polyfills": "^0.24.0"
   }
 }

--- a/templates/react-ts-example/package.json
+++ b/templates/react-ts-example/package.json
@@ -40,7 +40,7 @@
     "typescript": "^5.8.3",
     "typescript-eslint": "^8.35.0",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0"
+    "vite-plugin-node-polyfills": "^0.24.0"
   },
   "overrides": {
     "vite-plugin-node-polyfills": {

--- a/templates/react-ts-starter/package.json
+++ b/templates/react-ts-starter/package.json
@@ -38,6 +38,6 @@
     "typescript": "^5.8.3",
     "typescript-eslint": "^8.35.0",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0"
+    "vite-plugin-node-polyfills": "^0.24.0"
   }
 }

--- a/templates/vanilla-js-example/package.json
+++ b/templates/vanilla-js-example/package.json
@@ -20,7 +20,7 @@
     "prettier-plugin-tailwindcss": "^0.6.13",
     "tailwindcss": "^4.1.10",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0"
+    "vite-plugin-node-polyfills": "^0.24.0"
   },
   "dependencies": {
     "@junobuild/core": "^0.1.16",

--- a/templates/vue-example/package.json
+++ b/templates/vue-example/package.json
@@ -48,7 +48,7 @@
     "tailwindcss": "^4.1.10",
     "typescript": "~5.8.3",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0",
+    "vite-plugin-node-polyfills": "^0.24.0",
     "vite-plugin-vue-devtools": "^7.7.7",
     "vitest": "^3.2.4",
     "vue-tsc": "^2.2.10"

--- a/templates/vue-starter/package.json
+++ b/templates/vue-starter/package.json
@@ -47,7 +47,7 @@
     "tailwindcss": "^4.1.10",
     "typescript": "~5.8.3",
     "vite": "^6.3.5",
-    "vite-plugin-node-polyfills": "0.23.0",
+    "vite-plugin-node-polyfills": "^0.24.0",
     "vite-plugin-vue-devtools": "^7.7.7",
     "vitest": "^3.2.4",
     "vue-tsc": "^2.2.10"

